src/xdocs/config_naming.xml

```diff
@@ -2098,7 +2098,7 @@ class MyClass {
         <source>
 class MyClass {
   public MyClass() {} // OK
-  public void MyClass() {} // violation,  method Name 'MyClass' must not
+  public void MyClass() {} // violation, method Name 'MyClass' must not
```

Comment:
Do same update in MethodNameCheck.java

Reply:
https://github.com/checkstyle/checkstyle/blob/d2648e425bfbd9dfd8f1915a565591e6dba646aa/src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/MethodNameCheck.java#L146

Reply:
@7h3-3mp7y-m4n please always allow reviewer to resolve items.